cpp/order/Steinhardt.cc

```diff
@@ -303,17 +292,23 @@ float Steinhardt::normalize()
 std::complex<float> Steinhardt::normalizeWl()
 {
     // Get Wigner 3j coefficients:
-    // j1 from -l to l
-    // j2 from max(-l-j1, -l) to min(l-j1, l)
+    // j1 = j2 = j3 = l
+    // m1 from -l to l
+    // m2 from max(-l-m1, -l) to min(l-m1, l)
+    // m3 = -m1 - m2
```

Comment:
I rewrote this indexing math so it was clearer to anyone reading about Wigner 3j symbols and follows the conventions of these two pages much more clearly:
http://mathworld.wolfram.com/Wigner3j-Symbol.html
https://en.wikipedia.org/wiki/3-j_symbol